packages/calypso-products/src/plans-list.tsx

```diff
@@ -3609,3 +3619,20 @@ if ( isEnabled( 'plans/migration-trial' ) ) {
 		getTitle: () => i18n.translate( 'Business Trial' ),
 	};
 }
+
+if ( isEnabled( 'plans/hosting-trial' ) ) {
+	PLANS_LIST[ PLAN_HOSTING_TRIAL_MONTHLY ] = {
+		...getPlanBusinessDetails(),
+		type: TYPE_BUSINESS,
+		group: GROUP_WPCOM,
+		getProductId: () => 1058,
+		getPathSlug: () => PLAN_HOSTING_TRIAL_MONTHLY,
+		term: TERM_MONTHLY,
+		getBillingTimeFrame: () => i18n.translate( 'free trial' ),
+		getStoreSlug: () => PLAN_HOSTING_TRIAL_MONTHLY,
+		getTitle: () => i18n.translate( 'Hosting Trial' ),
```

Comment:
There's some discussion about the plan title here pet6gk-Ik-p2#comment-596, but not blocking. We can change this later.